src/components/AppDetailsToggle.vue

```diff
@@ -39,11 +39,10 @@ export default {
 	left: 0;
 	width: 44px;
 	height: 44px;
-	z-index: 149;
-	background-color: var(--color-main-background-darker);
+	z-index: 1051;
```

Comment:
🧐🧐🧐

Reply:
you dont like my 1051 index number? I can make it 2000 ;)